### ISSUE TITLE
Allow More Custom Tags

### ISSUE THREAD

**Issue by aterris** [apiaryio/heroku-datadog-drain-golang#67]
Looking through the code I noticed that you have limited custom tags in b6868b1323258d07e501082d90620b98b80c121e

I ran into this as I was trying to add some custom tags to some of my custom metrics. I was wondering if you could shed any light on the context behind that change?

I was thinking about doing one of the following...
1. Updating docs to reflect that
2. Remove the constraint entirely
3. Enable setting allowed custom tags via environment variable

Before I made a PR, I wanted to understand why you made that change in the first place.

**Comment by w-vi** [comment]
Yes they are limited because datadog does not handle very well if you have loads of them. This list should actually be configurable but apparently this has never happened. I this having the list defined by environment variable is a good approach so if you have a PR ready then please go ahead.

**Comment by aterris** [comment]
Not quite _ready_ but should be able to create one soon. Thanks!
